src/pages/Template/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { AppBar, Footer } from '../../components';
```

Comment:
We are able to import multiple components now because `/components` routes to `/components/index.js` which has all the exports for all files.